bin/templates/project/Activity.java

```diff
@@ -25,10 +25,9 @@ Licensed to the Apache Software Foundation (ASF) under one
 public class __ACTIVITY__ extends CordovaActivity
 {
     @Override
-    public void onCreate(Bundle savedInstanceState)
```

Comment:
CordovaActivity.onCreate has been called super.onCreate(savedInstanceState), seems we don't need it now.